code/modules/projectiles/guns/energy/blaster.dm

```diff
@@ -116,3 +116,61 @@
 		toggle_scope(2.0, usr)
 	else
 		to_chat(usr, "<span class='warning'>You can't look through the scope without stabilizing the rifle!</span>")
+
+
+/obj/item/gun/energy/disruptorpistol
+	name = "disruptor pistol"
+	desc = "Developed and produced by NanoTrasen for its internal security division, the NT DP-7 is a state of the art blaster pistol capable of firing reduced-power bolts which disrupt the central nervous system, inducing a stunning effect on the victim. It is also capable of firing full-power blaster bolts."
```

Comment:
this is a fluff_desc, descriptions shouldn't be this long

Reply:
Fixed